spec/lib/api/1.1/skus-spec.js

```diff
@@ -16,7 +17,9 @@ describe('Http.Api.Skus', function () {
         ]);
         return helper.startServer([
             dihelper.simpleWrapper(workflowApiService, 'Http.Services.Api.Workflows')
-        ]);
+        ]).then(function() {
+            _ = helper.injector.get('_');
```

Comment:
Lack the development doc is a gap. As whether we need to put it into on-core's global, I think that depends. If we see many unit-testing will use it I suggest to move it to on-core, this will sure bring much convenience. So you will not need to require `mocha`, `sinon`, `chai`, you use it without require because we do in the common helper.js. Of course you can reject this shortcut.

So this is just a suggestion for you, not an imperative command.